packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -188,35 +193,45 @@
 
 :global(.body__menu-open) .list-item {
   height: auto;
-  min-height: var(--item-height);
-  transition: max-height 0.15s ease-out;
+  min-height: var(--item-size);
   overflow: hidden;
+  width: 100%;
 }
 
-:global(.body__menu-open) .left-navigation,
-:global(.body__menu-open) .list-item {
+:global(.body__menu-open) .left-navigation {
   width: var(--width-left-navigation-when-expanded);
 }
 
 :global(.body__menu-open) .list-item .title {
   display: block;
   opacity: 1;
-  margin-left: var(--spacing-m);
+  margin-left: var(--spacing-s);
   /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-  color: var(--color-for-navbar-link);
+  color: var(--color-surface);
 }
+
 .list-item:not(.item__active):hover .icon > svg *:not([fill='none']) {
   /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
   fill: var(--color-for-navbar-icon-when-active);
 }
-.list-item .title:hover {
-  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
-  color: var(--color-for-navbar-link-when-hovered);
+
+.list-item:not(.item__active):hover .icon {
+  width: var(--icon-size-hover);
+  height: var(--icon-size-hover);
+  transition: all 0.25s ease-in-out;
+}
+
+.list-item:not(.item__active):hover .item-link {
+  /* TODO: add design tokens instead hard-coded values */
+  padding: 10px;
```

Comment:
Where exactly is the 10px padding in the design? Not sure if that is by accident or by purpose

Reply:
Hmm there I added this when I was testing transitions for links, there was some weird thing. But removed it for now, since transitions will be done separately. [cf7e148](https://github.com/commercetools/merchant-center-application-kit/pull/3203/commits/cf7e148140814b61b362f09bd1946856d3e71307)